src/ape_accounts/accounts.py

```diff
@@ -130,11 +132,51 @@ def delete(self):
         self.__decrypt_keyfile(passphrase)
         self.keyfile_path.unlink()
 
-    def sign_message(self, msg: SignableMessage) -> Optional[MessageSignature]:
-        user_approves = self.__autosign or click.confirm(f"{msg}\n\nSign: ")
-        if not user_approves:
+    def sign_message(self, msg: Any, **signer_options) -> Optional[MessageSignature]:
+        if not isinstance(msg, (SignableMessage, EIP712Message, str, int, bytes)):
+            logger.warning("Unsupported message type, (type=%r, msg=%r)", type(msg), msg)
             return None
 
+        user_approves = False
+
+        if isinstance(msg, str):
+            # Convert str to SignableMessage for handling below
+            user_approves = self.__autosign or click.confirm(f"Message: {msg}\n\nSign: ")
+            msg = encode_defunct(text=msg)
+        elif isinstance(msg, int):
+            user_approves = self.__autosign or click.confirm(f"Message: {msg}\n\nSign: ")
+            msg = encode_defunct(hexstr=HexBytes(msg).hex())
+        elif isinstance(msg, EIP712Message):
```

Comment:
you want this to fall through to signable message
```suggestion

        if isinstance(msg, EIP712Message):
```